lib/pages/feed/item_list.dart

```diff
@@ -0,0 +1,172 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:intl/intl.dart';
+
+import '../../common/colors.dart';
+import '../../services/pixel_service.dart';
+import '../../models/pixel.dart';
+import '../../models/comment.dart';
+import '../../widgets/comment/comment_list.dart';
+
+const containerMargin = EdgeInsets.only(left: 10, right: 10, top: 13, bottom: 13);
+
+class ItemList extends StatelessWidget {
```

Comment:
`ListItem` or `FeedItem` might be a better name as this class represents an item from a list, not a list from an item.